lib/api/src/sys/imports.rs

```diff
@@ -121,6 +123,37 @@ impl Imports {
             .insert((ns.to_string(), name.to_string()), val.into());
     }
 
+    /// Imports (any) shared memory into the imports.
+    /// (if the module does not import memory then this function is ignored)
+    #[cfg(feature = "compiler")]
```

Comment:
I find strange to tie this function to the "compiler" feature, as it looks like a Runtime one.

Reply:
This is the same on master.